recipes/open62541/all/submoduledata.yml

```diff
@@ -0,0 +1,11 @@
+submodules:
```

Comment:
Huge improvement over the other mechanism! Let's see if the CCI team approves!

Reply:
I agree, I love that part from @syoliver code